build.sbt

```diff
@@ -111,95 +259,118 @@ lazy val core = libraryProject("core")
     ),
     buildInfoPackage := organization.value,
     libraryDependencies ++= Seq(
-      caseInsensitive,
-      catsCore,
-      catsEffectStd,
-      catsParse.exclude("org.typelevel", "cats-core_2.13"),
-      fs2Core,
-      fs2Io,
-      ip4sCore,
-      literally,
-      log4s,
-      munit % Test,
-      scodecBits,
+      caseInsensitive.value,
+      catsCore.value,
+      catsEffectStd.value,
+      catsParse.value.exclude("org.typelevel", "cats-core_2.13"),
+      fs2Core.value,
+      ip4sCore.value,
+      literally.value,
+      log4s.value,
+      munit.value % Test,
+      scodecBits.value,
       slf4jApi, // residual dependency from macros
-      vault,
+      vault.value
     ),
     libraryDependencies ++= {
       if (isDotty.value) Seq.empty
-      else Seq(
-        scalaReflect(scalaVersion.value) % Provided
-      )
+      else
+        Seq(
+          scalaReflect(scalaVersion.value) % Provided
+        )
     },
-    unusedCompileDependenciesFilter -= moduleFilter("org.scala-lang", "scala-reflect"),
+    unusedCompileDependenciesFilter -= moduleFilter("org.scala-lang", "scala-reflect")
   )
+  .jvmSettings(
+    libraryDependencies += fs2Io.value
+  )
+  .jsSettings(
+    libraryDependencies += scalaJavaTime.value,
+    scalacOptions ~= { _.filterNot(_ == "-Xfatal-warnings") }
+  )
+  .jsConfigure(_.disablePlugins(DoctestPlugin))
```

Comment:
:thinking: I was hoping to start leaning harder into these.  I guess most of the value is testing them on at least one platform.

Reply:
I was mostly getting into trouble with warnings for Scala.js facades, which have `js.native` definitions that made the compiler upset. I think I tried `@nowarn`ing them but then it was scala 2.12 vs 2.13, in the end this was a compromise. Happy to take another look.

Reply:
No, not a priority.  It only gets dicey if our examples become our only test coverage.  We're more likely to use it to ensure that our examples are correct.  Closer to an mdoc than a unit test.